From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sut, 22 Apr 2017 18:01:00 +0200
Subject: [PATCH] Do not install METIS

METIS is optional (and not needed for Julia), but `make install` fails when
it's not built, so we must disable it in Makefile.
--- a/Makefile
+++ b/Makefile
@@ -34,7 +34,7 @@
 
 # install all packages in /usr/local/lib and /usr/local/include
 # (note that CSparse is not installed; CXSparse is installed instead)
-install: metisinstall
+install:
 	( cd SuiteSparse_config && $(MAKE) install )
 	( cd AMD && $(MAKE) install )
 	( cd BTF && $(MAKE) install )
